documentation/misc/gas_balancing.md

```diff
@@ -0,0 +1,54 @@
+## Gas Balancing
+
+The gas balancing process targets to set gas costs of syscalls to be in line with
+10 gas per nanosecond on reference hardware.
+The process can be either performed for all syscalls based on existing messages and chain or targeted
+at single syscall.
+
+#### Reference hardware
+
+The reference hardware is TR3970x with 128GB of RAM. This is what was available at the time and
+may be subject to change.
+
+### Complete gas balancing
+
+Complete gas balancing is performed using `lotus-bench` the process is based on importing a chain export
+and collecting gas traces which are later aggregated.
+
+Before building `lotus-bench` make sure `EnableGasTracing` in `chain/vm/runtime.go` is set to `true`.
+
+The process can be started using `./lotus-bench import` with `--car` flag set to the location of
+CAR chain export. `--start-epoch` and `--end-epoch` can be used to to limit the range of epochs to run
```

Comment:
Will users know where to get a CAR chain export?

Reply:
I'm assuming it will be a developer reading these docs, so I assume they will know.